akka-docs/src/test/java/jdocs/stream/KillSwitchDocTest.java

```diff
@@ -52,7 +52,7 @@ public void uniqueKillSwitchShutdownExample() throws Exception {
     //#unique-shutdown
     final Source<Integer, NotUsed> countingSrc =
       Source.from(new ArrayList<>(Arrays.asList(1, 2, 3, 4)))
-        .delay(FiniteDuration.apply(1, TimeUnit.SECONDS), DelayOverflowStrategy.backpressure());
+        .delay(      java.time.Duration.ofSeconds(1), DelayOverflowStrategy.backpressure());
```

Comment:
Spurious spaces

Reply:
We need a Java formatting tool which hooked into SBT:)

Reply:
Yes #20891 :)